### PR TITLE
searchUser, addUser, vote refine

### PR DESCRIPTION
**Pull request by Talos6**
1. Create SearchBar that is able to search user to be added. User who already in the group will not be added.
2. addUser function. Add a new user to the group and update the group vote count for the user's own songs. 
3. refine vote count to show both upvote and downvote count. Using different colour to distinguish
4. simple styling on searchUser bar

### CODE REVIEW

File: sound-bubble/imports/ui/components/Groups/Search.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        };
+    }
+
+    onInputChange(event){
+      this.searchUser(event.target.value);
+    }
+
+    searchUser(userName){
+      const foundUsers = this.props.allUsers.filter(user => user.profile.display_name.toLowerCase().includes(userName.toLowerCase()));
+      console.log(foundUsers);
```
Comment (martipal) [inline sound-bubble/imports/ui/components/Groups/Search.jsx]: this console.log could be removed

File: sound-bubble/imports/ui/components/Home/Vote.jsx [hunk 2]
```diff
@@ -25,7 +25,9 @@ class Vote extends React.Component {
               <span className="tooltiptext">Undo Upvote</span>
             </div>
           </div>
```
Comment (martipal) [inline sound-bubble/imports/ui/components/Home/Vote.jsx]: Ah, I restyled the vote behaviour in my styling branch as well. I guess we can pick which one we want.

Comment (Talos6) [reply sound-bubble/imports/ui/components/Home/Vote.jsx]: alright, I just using different colour to distinguish between upvote amount and downvote amount

File: sound-bubble/imports/ui/components/Groups/Search.jsx (hunk 1 above)
Comment (Talos6) [reply sound-bubble/imports/ui/components/Groups/Search.jsx]: Just removed,

File: sound-bubble/imports/ui/components/Home/Vote.jsx (hunk 2 above)
Comment (veronicz) [reply sound-bubble/imports/ui/components/Home/Vote.jsx]: I think Martin had the colours too lol. One vote for keeping Martin's change.

**Comment by Talos6** [comment]
Yeah, I see after left or delete a group, the current group should also been changed

**Pull request merged**